src/signature_pad.ts

```diff
@@ -87,12 +128,34 @@ export default class SignaturePad extends EventTarget {
       : SignaturePad.prototype._strokeUpdate;
     this._ctx = canvas.getContext('2d') as CanvasRenderingContext2D;
 
+    this._et = new SignatureEventTarget(options.documentAsEventTarget);
+
     this.clear();
 
     // Enable mouse and touch event handlers
     this.on();
   }
 
+  addEventListener(
+    type: string,
+    listener: EventListenerOrEventListenerObject | null,
+    options?: boolean | AddEventListenerOptions,
+  ): void {
+    this._et.addEventListener(type, listener, options);
+  }
+
+  dispatchEvent(event: Event): boolean {
+    return this._et.dispatchEvent(event);
+  }
+
+  removeEventListener(
+    type: string,
+    callback: EventListenerOrEventListenerObject | null,
+    options?: boolean | EventListenerOptions,
+  ): void {
+    return this._et.removeEventListener(type, callback, options);
+  }
+
```

Comment:
It looks like these methods are copied from `SignatureEventTarget` wouldn't it just be easier for `SignaturePad` to extend `SignatureEventTarget`?

Reply:
yes, you're right. Using composition over inheritance here won't get better advantage. There wouldn't be other `EventTarget` behaviour, would it? However when it does need it or this class needs to inherit other class, then perhaps we could implements the `SignatureEventTarget`, instead of extends it.

Reply:
ah yeah, i'll change this in my next commit.